index.html

```diff
@@ -0,0 +1,128 @@
+  <main>
+    <section class="headline">
+      <h1><span class="hello">"Hello! Engineers"</span><br> <strong>ANNUAL GENERAL<br> CONFERENCE<br> 2024</strong></h1>
+      <p class="intro">This conference will provide a common platform for all practitioners and affiliates from diverse engineering and related backgrounds to share ideas, experiences, and network.</p>
+      <p class="date"><strong>2024.10.17(THU) ~ 16(FRI)</strong> </p>
+      <P class="location">@ Accra International Conference Center</P>
+    </section>
+
+    <section class="program">
+      <h2 class="program">Main Program</h2>
+      <hr>
+      <div class="desktop-flex-container">
+        <div class="program-item">
+          <i class="material-symbols-rounded">podium</i>
+          <span class="icon">Lecture</span>
+          <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+        </div>
+        <div class="program-item">
+          <i class="material-symbols-rounded">photo_frame</i>
+          <span class="icon">Exhibition</span>
+          <p>Appreciate various creations applying GhiE standards.</p>
+        </div>
+        <div class="program-item">
+          <i class="material-symbols-rounded">forum</i>
+          <span class="icon">Forum</span>
+          <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div class="program-item">
+          <i class="material-symbols-rounded">home_repair_service</i>
+          <span class="icon">Workshop</span>
+          <p>Try creating your own work using open source license rather than just watching.</p>
+        </div>
+        <div class="program-item">
+          <i class="material-symbols-rounded">hub</i>
+          <span class="icon">Network</span>
+          <p>Get opportunities to network with other Engineers around the world.</p>
+        </div>
+      </div>
+      <button class="join">Join</button>
+      <a href="#" class="whole-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="speakers" class="speakers">
+      <!-- <h2>Featured Speakers</h2>
+      <hr>
+      <div class="speaker-flex" >
+
+        <div class="speaker-container">
+          <div class="speaker-image-container"><img src="images/sophia.jpg" alt=""><img class="checkered" src="checkered" alt=""></div>
+          <div class="speaker-text">
+            <h3>Harriet Oteng</h3>
+            <p class="position"><i>Lead Engineer of Quartz Dam Construction</i> </p>
+            <hr class="gray">
+            <p class="interest">Improve member benefits; monitor achievements within the council year and suggest other business friendly ways to Council to improve our collaboration with the public</p>
+          </div>
+        </div>
+
+      </div>
+
+      <button id="more" class="more">
+        <span class="more-text" >MORE</span>
+        <span class="fa fa-solid fa-chevron-down more-icon"></span>
+      </button> -->
```

Comment:
- Hey, please, kindly remove this commented-out piece of code as this makes your codebase not clean and professional.